### PR TITLE
re-commit the changes

### PR DESCRIPTION
**Pull request by zldzksk1**
I have noticed that I didn't commit the latest version of my work, so I submitted a new pull request. I am sorry for any inconvenience you guys might have.

### CODE REVIEW

File: client/src/components/layouts/Dashboard.js [hunk 1]
```diff
@@ -4,21 +4,35 @@ import{NavLink as RouterNavLink} from 'react-router-dom';
...
+                    <a href="#">  Manage Account</a> 
+                </div>
+                <div className='dash-card-cell'>
+                    <a href="ContactUs"> Customer Service</a> 
+                </div>
+                <div className='dash-card-cell'>
+                    <a href="#"> <img src = {require("../../assets/logo_small_two.png")} className='img-fluid shadow-4'/></a> 
+                </div>
+                <div>
+                   All copyrights reserved, quizbanana.,llc 2022
```
Comment (pamelayin) [inline client/src/components/layouts/Dashboard.js:33]: not important right now but it's possible to do 2022 as dynamic. please refer here https://updateyourfooter.com/

File: client/src/index.css [hunk 2]
```diff
@@ -11,3 +11,36 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
     monospace;
 }
+
```
Comment (pamelayin) [inline client/src/index.css:14]: wouldn't it be better to rename this file as App.css?

**Review by pamelayin — APPROVED** [review]
Please see the comments. They are minor things, just suggestions, doesn't have to be fixed.

File: client/src/components/layouts/Dashboard.js (hunk 1 above)
Comment (zldzksk1) [reply client/src/components/layouts/Dashboard.js:33]: I will do!

**Pull request merged**